.github/workflows/sync-fork.yaml

```diff
@@ -0,0 +1,24 @@
+name: Sync with Original Repository
+
+on:
+  schedule:
+    - cron: '0 12 * * *' # Run every day at 12:00
+  workflow_dispatch: # Allows manual trigger
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout forked repo
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0 # Fetch all history
+
+    - name: Fetch original repo
+      run: git remote add original https://github.com/hms-dbmi-cellenics/worker.git
+
+    - name: Reset to original repo's state
+      run: |
+        git fetch original
+        git reset --hard original/master
+        git push --force
```

Comment:
Can we use something like this instead? https://github.com/marketplace/actions/fork-sync